src/libs/import.c

```diff
@@ -721,8 +731,10 @@ static void _lib_import_update_preview(GtkFileChooser *file_chooser, gpointer da
       g_object_unref(pixbuf);
       pixbuf = tmp;
     }
+
+    have_preview = TRUE;
   }
-  if(no_preview_fallback || !have_preview)
+  if(!have_preview && no_preview_fallback)
```

Comment:
When not having a preview (`have_preview == FALSE`) we have to go into this, no matter what `no_preview_fallback` says.